spec.html

```diff
@@ -3955,6 +4810,118 @@ <h1>ToUint8Clamp ( _argument_ )</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-to-bigint" aoid="ToBigInt">
+      <h1>ToBigInt ( _argument_ )</h1>
+      <p>The abstract operation ToBigInt converts its argument _argument_ to a BigInt value, or throws if an implicit conversion from Number would be required.</p>
+      <emu-alg>
+        1. Let _prim_ be ? ToPrimitive(_argument_, hint Number).
+        1. Return the value that _prim_ corresponds to in <emu-xref href="#table-to-bigint"></emu-xref>.
+      </emu-alg>
+      <emu-table id="table-to-bigint" caption="BigInt Conversions">
+        <table>
+          <tbody>
+            <tr>
+              <th>
+                Argument Type
+              </th>
+              <th>
+                Result
+              </th>
+            </tr>
+            <tr>
+              <td>
+                Undefined
+              </td>
+              <td>
+                Throw a *TypeError* exception.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                Null
+              </td>
+              <td>
+                Throw a *TypeError* exception.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                Boolean
+              </td>
+              <td>
+                Return `1n` if _prim_ is *true* and `0n` if _prim_ is *false*.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                BigInt
+              </td>
+              <td>
+                Return _prim_.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                Number
+              </td>
+              <td>
+                Throw a *TypeError* exception.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                String
+              </td>
+              <td>
+                <emu-alg>
+                  1. Let _n_ be StringToBigInt(_prim_).
+                  1. If _n_ is *NaN*, throw a *SyntaxError* exception.
```

Comment:
Should this use `SameValue(_n_, *NaN*)`? i don't feel strongly either way.

Reply:
I think using `SameValue(_n_, *NaN*)` here would be a unnecessary complex step, but I can change if you think it is better.